pkg/plugin/aggregation/run.go

```diff
@@ -154,3 +167,32 @@ func Cleanup(client kubernetes.Interface, plugins []plugin.Interface) {
 		p.Cleanup(client)
 	}
 }
+
+func sendUpdates(updater *status.Updater, ag *Aggregator) {
```

Comment:
This feels like a pretty important function floating around. It looks like it might make sense to live on the aggregator struct or on the updater struct. I think it would be ok for now. If you think it makes sense to find a home for this, maybe add a TODO. If I'm just over thinking it feel free to ignore